src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
the calendar should have modifiers according to task requirements
```suggestion
    <div class="calendar calendar__start-day--sat calendar__month-length--29">
```

and the result with these modifiers should be like that

<img width="476" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/102411130/089dc45c-709a-4942-91b8-9aa275610fdc">

Reply:
if I change `start-day`, it should change margin left for the first day in the moth.
if I change `month-length`, number of days in the month should change

Reply:
check task requirements
<img width="743" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/102411130/e643db45-c1a3-43b1-bae0-f9b1422a4000">